src/farmer.rs

```diff
@@ -344,7 +335,7 @@ pub struct Farmer {
     cmd_sender: Arc<Mutex<Option<oneshot::Sender<oneshot::Sender<()>>>>>,
     reward_address: PublicKey,
     plot_info: Arc<HashMap<PathBuf, Plot>>,
-    node: Node,
+    _node: Node,
```

Comment:
why are we even returning `node` inside the `Farmer` struct?
and also, if it is necessary, why is it underscored?

Reply:
I can remove it at all, but I believe we can use it in future.